darwin/exporter/formats/mask.py

```diff
@@ -561,11 +556,9 @@ def offset_complex_polygon(polygons: List, offset_x: int, offset_y: int) -> List
         new_polygons.append(offset_simple_polygon(polygon, offset_x, offset_y))
     return new_polygons
 
+
 def offset_simple_polygon(polygon: List, offset_x: int, offset_y: int) -> List:
     new_polygon = []
     for point in polygon:
-        new_polygon.append({
-            'x': point['x'] + offset_x,
-            'y': point['y'] + offset_y
-        })
-    return new_polygon
\ No newline at end of file
+        new_polygon.append({"x": point["x"] + offset_x, "y": point["y"] + offset_y})
```

Comment:
This could be a listcomp I guess, and would be a bit faster as one, but is likely more readable this way, so only change it if you want to.